resources/translations/passwords-evolved.pot

```diff
@@ -9,6 +9,9 @@ msgstr ""
 "Language-Team: Carl Alexander <contact@carlalexander.ca>\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
+msgid "Passwords Evolved requires PHP 5.3.7 or higher. Your WordPress site is using PHP %s"
+msgstr ""
+
```

Comment:
I'd remove this unless you think we can translate it. But I don't think our translation files will be loaded at that point.